src/components/Artist/Artist.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useFetchReleases } from '../../services/hooks';
+import withList from '../HOCs/WithList/WithList';
+import ReleaseItem from '../Release/ReleaseItem';
+import { useParams } from 'react-router-dom';
+import style from './Artist.css';
+
+const Artist = () => {
+  const { artist, artistId } = useParams();
```

Comment:
Because this component is the direct descendant of a Route, you have access to match and can grab params from there without needing to use useParams. Just give the Artist component `{ match }` on line 8, and then you can use match.params.artist and match.params.artistId.